### ISSUE TITLE
Add “send all” support

### ISSUE THREAD

**Issue by sublayerio** [Berry-Pool/nami-wallet#23]
Make it more convenient for people to empty their wallet in a single transaction without having to figure out the exact decimals given the tx fee.

**Comment by pascallapointe** [comment]
Yes this feature is missing, while being simple to implement for small wallet, it is a little more complicated for large native assets holder as a complete balance can't be sent in one single transaction.
I'll be working on it soon.

**Comment by sublayerio** [comment]
Sounds like a sophisticated implementation. Do you have to batch it into multiple txs and allocate enough tx fees?

**Comment by pascallapointe** [comment]
Yes, each tx would require the fees + meet the minimum ada requirement for assets. In a perfect world, we would split it evenly into different transactions to create balanced utxo on the receiving end.
